docs/admin/kubelet.md

```diff
@@ -106,7 +106,6 @@ kubelet
       --kubeconfig="/var/lib/kubelet/kubeconfig": Path to a kubeconfig file, specifying how to authenticate to API server (the master location is set by the api-servers flag).
       --kubelet-cgroups="": Optional absolute name of cgroups to create and run the Kubelet in.
       --lock-file="": <Warning: Alpha feature> The path to file for kubelet to use as a lock file.
-      --log-flush-frequency=5s: Maximum number of seconds between log flushes
```

Comment:
Whenever I removed `util/logs.go` and move it to sub-dir, this flag will be removed by updating generated docs.

I think it has relationship with `https://github.com/resouer/kubernetes/blob/log-dev/pkg/util/log/logs.go#L30`, but not sure of the root cause ...